source/utilities.tex

```diff
@@ -11676,9 +11685,11 @@
 \pnum
 \constraints
 \tcode{\&F::operator()} is well-formed when treated as an unevaluated operand and
-\tcode{decltype(\brk{}\&F::operator())} is of the form
+\tcode{decltype(\brk{}\&F::operator())} is either of the form
 \tcode{R(G::*)(A...)}~\cv{}~\tcode{\opt{\&}~\opt{noexcept}}
-for a class type \tcode{G}.
+or of the form
+\tcode{R(*)(G, A...)}~\tcode{\opt{noexcept}}
```

Comment:
I think closing the `tcode` and reopening it was only needed for the _cv_ macro, which isn't in this new text.

```suggestion
\tcode{R(*)(G, A...) \opt{noexcept}}
```

Reply:
fixed